tests/sender/send_data.py

```diff
@@ -315,3 +322,228 @@ def test_sender_as_handler_static(self):
             con.close()
         except Exception as error:
             self.fail("Problems with test: %s" % str(error))
+
+    def test_config_files_path_standard_case(self):
+        """
+        Test that verifies that a correct path for the
+        configuration file is detected.
+        """
+
+        engine_config = SenderConfigSSL(address=(self.server, self.port),
+                                        key=self.key,
+                                        cert=self.cert,
+                                        chain=self.chain,
+                                        check_hostname=False,
+                                        verify_mode=CERT_NONE,
+                                        verify_config=True)
+        result = Sender.check_config_files_path(engine_config)
+        self.assertEqual(result, True)
```

Comment:
**suggestion(non-blocking):** can you please use the `assertTrue` method instead? Or are you using `assertEqual` for any special reason?